.github/workflows/docker.yml

```diff
@@ -0,0 +1,42 @@
+name: Docker Build & Push Simapp (main)
+# Build & Push builds the simapp docker image on every push to main and
+# and pushes the image to https://ghcr.io/cosmos/ibc-go-simd
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**.go'
```

Comment:
one thing I would double check is that we should also build and push when this file changes, we could also add `on: workflow_dispatch` just to make it easy to manually trigger a build from main